src/e2e-test/java/io/cdap/plugin/datastore/stepsdesign/DataStoreActions.java

```diff
@@ -0,0 +1,20 @@
+package io.cdap.plugin.datastore.stepsdesign;
```

Comment:
Please add the license and java doc. Refer any existing action class.

Reply:
Execute the tests with verify command to for check styles locally.

Reply:
added the  javadoc and separate directories.